templates/identity/email-password/login.html.ejs

```diff
@@ -1,15 +1,17 @@
 <h1>Log in</h1>
-<form action="/idp/login" method="post">
-  <% if (errorMessage) { %>
-    <p class="error"><%= errorMessage %></p>
+<form method="post">
+  <% const safePrefilled = locals.prefilled || {}; %>
```

Comment:
Argh, I guess we can't use `.?` below because Node 12.

Reply:
I think we can just override `prefilled` though:

```suggestion
  <% prefilled = locals.prefilled || {}; %>
```

Reply:
> Argh, I guess we can't use `.?` below because Node 12.

Yea was quite annoying. Had that first but then the CI tests started failing.